tasks/upgrade_component.yml

```diff
@@ -14,9 +14,9 @@
     src: "{{ item }}"
     remote_src: true
     dest: "/tmp/upgrade/{{ service_name }}/{{ item | basename }}-{{timestamp}}"
+    owner: "{{ config_owner }}"
```

Comment:
Why this change of variable name?

Reply:
I didn't like the variable being the same as the argument